### PR TITLE
Add San Diego file

### LINKED ISSUE #8: sarah-schroeder tracks progress
- [x] create a feature branch
- [x] add some commits
- [x] open a pull request
- [x] collaborate and add additional commits
- [x] merge my pull request with 1 :+1:

### PR DESCRIPTION
**Pull request by sarah-schroeder**
This pull request adds the San Diego file to the project.  This PR closes #8 

Some questions:
1. If you've been to SD before, what did you like?  Not like?
2. If you have kids, what might be of interest to you?

### CODE REVIEW

File: sarah-schroeder-sd.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# San Diego, CA
+
+### Restaurants
+- Tiger Tiger
```
Comment (hollenberry) [inline sarah-schroeder-sd.md]: 👍 This place is great! I actually live just around the corner!

File: sarah-schroeder-sd.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- Oscar's Fish Tacos
```
Comment (hollenberry) [inline sarah-schroeder-sd.md]: @sarah-schroeder Where's Oscar's? I haven't heard of this one. 😄  Can you add a taco recommendation here?

File: sarah-schroeder-sd.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+### Local Attractions
+- Sunset Cliffs
```
Comment (alanzhong) [inline sarah-schroeder-sd.md:8]: :O

**Comment by sarah-schroeder** [comment]
@hollenberry take a look at f354b24

**Pull request merged**